packages/blade/src/components/Icons/_Svg/Path/Path.d.ts

```diff
@@ -1,3 +1,7 @@
+export type PreventDashedProps<T> = {
```

Comment:
Optional: Maybe `NonDashedProps` can be better term for this

Reply:
Better. 👍